app/views/location_admins/edit.html.erb

```diff
@@ -0,0 +1,22 @@
+<h2>Edit <%= resource_name.to_s.humanize %></h2>
+
```

Comment:
All the views associated with locations_admins are not used as of now.  It can be used in the future for allowing donors to log in and generate their own receipts